### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by jackvolonte** [day22/Splendor#1]
The GUI seems a little cluttered, but the game is really complex so I could see why. Besides that this project looks great so far

**Comment by njpon** [comment]
I agree that the GUI could look a little cleaner. Perhaps make the icons a little smaller so it looks a little more organized?
